_posts/2019-03-04-jumping-on-the-dash-less-bandwagon.markdown

```diff
@@ -0,0 +1,64 @@
+
+Next, with all this infrastructure provided to you, gone are hours or days of writing boilerplate code. All the lambda functions I’ve written listen for an incoming event (http, file, database, whatever) and consume it. No setting up scaffolding code. No writing complex convoluted setup logic for unit tests. Just the minimum code to consume the event and suite of testing scenarios to verify the behaviours that I expect. [Single Responsibility Principle](https://en.wikipedia.org/wiki/Single_responsibility_principle) at its finest.
+
+Time to think in terms of events and states as ultimately you’re responding to state change events within the bounds of a fully distributed system. This requires a more disciplined focus on what triggers these events and how things can, and will, [go awry at any time](https://en.wikipedia.org/wiki/Chaos_engineering).
+
+The architecture diagram below for a photo management service, on a recent project, demonstrates just how powerful Serverless can be. The service allows end-users to take, upload and manage maintenance photos of the organisation's physical equipment in the field. To deliver this, nothing more beyond CloudFormation scripts and Lambda code was needed.
+
+<img style="display: block; margin-left: auto; margin-right: auto;" alt="photo management service" src="{{ site.url }}/img/bandwagonless/photo-app.png"/>
+
+**Link of Interest:** [6 things I’ve learned in my first 6 months using serverless](https://read.acloud.guru/six-months-of-serverless-lessons-learned-f6da86a73526)
+
+## Wrapping Up
+
+For years now the phrase “start-ups are coming to eat your lunch” has been shouted from all corners of the Internet. This warning hasn’t changed, but what has is the extremely low barrier of entry to eat it. A few years ago, start-ups were still paying for virtual worlds that mirrored the physical; EC2 servers, VPC networking, relational databases designed just like on premise, VPN links back to data centres, etc. Dedicated people were also required (and paid) to build, manage and operate all these things.
+
+Today with serverless, entire development-to-production environments can be scripted and deployed by a single developer. The literal cost? A few cents at most. In fact, your time spent reading this post has probably cost more to the company you work for than the yearly operating costs of the serverless systems I’ve worked on.
+
+Serverless is a relentless freight train building that is quickly building up momentum. You can either get onboard or be crushed. Join me in my next post where I discuss the new SDLC; Serverless Development Life Cycle.
```

Comment:
You just left an absolute bombshell in the last paragraph, and is an excellent way to finish.

Either move this paragraph higher up, or remove it.

The last paragraph is the one that will stick in the mind.  That is sumed up beautifully in the previous paragraph - don't lose it.